recipes/elfio/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
+set_target_properties(example PROPERTIES
+    CXX_STANDARD 11 # Elfio requires at least c++11
+    CXX_STANDARD_REQUIRED ON)
```

Comment:
Should this check also be in the recipe?

Reply:
don't think so, it is not building anything. C++11 is meanwhile the default, and for those who run into build issues, they will get a very helpful message anyway. 
So I think it's better to keep the recipe simple

Reply:
How does the consumer know it's a requirement? What if I have `settings.compiler.cppstd=03` for my project? There's no indication it might be a problem 🤔

Reply:
you will get a very descriptive error message when you compile your program. and you will deserve it ;-) 

you can not have this check in the recipe, you can only check if the compiler in the profile is 'modern' enough to have C++11 support.
At some point in time you need to be able to assume a modern (10+ years old) compiler that has C++11 support is used, and imho this point is already here ;-)